jetty-server/src/main/java/org/eclipse/jetty/server/handler/AllowSymLinkAliasChecker.java

```diff
@@ -33,7 +33,9 @@
  * to check resources that are aliased to other locations.   The checker uses the
  * Java {@link Files#readSymbolicLink(Path)} and {@link Path#toRealPath(java.nio.file.LinkOption...)}
  * APIs to check if a file is aliased with symbolic links.</p>
+ * @deprecated use {@link org.eclipse.jetty.server.SymlinkAllowedResourceAliasChecker} instead.
  */
+@Deprecated
```

Comment:
Perhaps replace the actual implementation with `SymlinkAllowedResourceAliasChecker`

Reply:
They have different constructors. I could possibly combine them and add a new constructor to this class and deprecate the old one.

Reply:
Ah yes.... would have to make it a lifecycle and look for the threadload context in doStart.  Possible, but not worth it.